test/models/lockable_test.rb

```diff
@@ -110,6 +110,30 @@ def setup
     assert_equal 0, user.reload.failed_attempts
   end
 
+  test "unlocks a user keeping its last unlock_token when configured" do
```

Comment:
```suggestion
  test "unlocks an user keeping its last unlock_token when configured" do
```

Reply:
That's an interesting thing in the English language!

In this case, _a_ user is correct, as the a/an usage depends on the _sound_ of the next word. It's pronounced something like _yuzer_, so it has a consonant initial sound.

Reference: https://forum.wordreference.com/threads/a-an-user.480322/#:~:text=A%20user%20is%20correct.,vowel%20as%20the%20first%20letter.